RELEASING.md

```diff
@@ -0,0 +1,12 @@
+Release procedure for dependency-analysis-android-gradle-plugin
```

Comment:
I think this also needs to include "run the tests and ensure they pass." I will probably add a `dependsOn` to the release task that requires the tests to be green.